app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion "23.0.3"
```

Comment:
nice when this is done as a separate PR. Usually on bigger projects updating the tools can make ppl giving your PR a +1 take a whoooole lot longer, so just safer to split for faster merges